libretroshare/src/ft/ftcontroller.cc

```diff
@@ -1732,13 +1624,12 @@ void    ftController::statusChange(const std::list<pqiServicePeer> &plist)
     std::map<RsFileHash, ftFileControl*>::iterator it;
 	std::list<pqiServicePeer>::const_iterator pit;
 
-#ifdef CONTROL_DEBUG
+//#ifdef CONTROL_DEBUG
```

Comment:
anxious to try, but i don't know how to put in my fork of rs https://github.com/felisucoibi/RetroGhibli